src/main/java/dinkplugin/notifiers/LeaguesNotifier.java

```diff
@@ -261,7 +309,7 @@ private static String ith(int i) {
         thresholds.put(56_000, "Dragon");
         TROPHY_BY_POINTS = Collections.unmodifiableNavigableMap(thresholds);
 
-        TIER_BY_RELIC = Map.ofEntries(
+        TIER_BY_RELIC = new HashMap<>(Map.ofEntries(
```

Comment:
this can be an empty map now, or we can keep these entries in case the cache enums/structs significantly change (there is no harm in keeping the values)

with this pr, if new relics are added to any tier, or if the relic names change, we will still handle relic unlocks gracefully

Reply:
in rare circumstances, runelite is weird, so we shouldn't switch to an empty map

https://discord.com/channels/301497432909414422/419891709883973642/1177706719103680613